Makefile

```diff
@@ -8,6 +8,11 @@ SNAPSHOT?=yes
 PYTHON_ENV?=${BUILD_DIR}/python-env
 VIRTUALENV_PARAMS?=
 FIND=find . -type f -not -path "*/vendor/*" -not -path "*/build/*" -not -path "*/.git/*"
+GOLINT=golint
```

Comment:
my fault, I assume this lint target calls all the other lint targets like we do for update and other things.